src/utils/money.cljs

```diff
@@ -60,7 +60,9 @@
 (defn equal-to
   [n1 n2]
   (when-let [^js bn1 (bignumber n1)]
-    (.eq ^js bn1 n2)))
+    (try
+      (.eq ^js bn1 n2)
+      (catch :default _ false))))
```

Comment:
Here we're attempting to prevent exceptions from being thrown because of the BigNumber library failing to do a comparison. Perhaps we should not catch all exceptions here, but I think it's safer to assume that any failure would mean the comparison failed, which would mean the two things are not equal.

But maybe there's a better way to handle these exceptions, thoughts?

Reply:
@seanstrom, I think that we can avoid try/catch which is not common in our codebase by checking both arguments with `(bignumber)` before calling `.eq`, wdyt?

Reply:
I am not sure what is the best approach, but I've recently updated `greater-than` `less-than` `greater-than-or-equals` functions with a `bignumber?` check. The check was only needed on the first argument to avoid the error. We could use the same approach on `equal-to` for consistency.

Reply:
@briansztamfater, I also think that everything should be wrapped in checks, including second argument, because for `equal-to` second argument causes exception. I wrote an example down below.